[all hosts] (Visual Studio) Revise VS JS->TS steps (#4868)
OfficeDev/office-js-docs-pr: docs/develop/convert-javascript-to-typescript.md

    ```

1. Save the file.
1. Open the npm project properties by going to **Tools** > **Options**, then **Projects and Solutions** > **Web Package Management** > **Package Restore**. Set both **Restore On Project Open** and **Restore On Save** to "True". Select **OK** to save the settings.

### Update the JavaScript files

Change your JavaScript files (**.js**) to TypeScript files (**.ts**). Then, make the necessary changes for them to compile. This section walks through the default files in a new project.

1. Find the **Home.js** file and rename it to **Home.ts**.

    > [!WARNING]

>     > There is currently a issue in template's **Home.js** file: the reference on line 95 to `textvalue` should be changed to `textValue`.

Reviewer: ```suggestion
    > There's currently an issue in the template's **Home.js** file: the reference on line 95 to `textvalue` should be changed to `textValue`.
```
